nominees/social-income.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Social Income",
+  "aliases": [
+    ""
+  ],
+  "description": "A open source initiative to fight SDG 1 with direct cash transfers for people in need",
```

Comment:
Updated description: Social Income provides an end-to-end solution to manage direct-cash transfer projects or universal basic income pilots. This includes funding grassroots fundraising, contributor management, selection management with third-party NGOs and payment management (not the actual payment, this can be done with any local mobile money provider).